templates/reset_password.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<div class="row justify-content-center"> 
+    <div class="col-md-7 text-center">
+        <h2 class="mt-4">Enter new password</h2>
+        <form method="POST">
```

Comment:
An alternative that has to be mentioned is to do the call async. Avoiding any page refreshes and you can confirm password matching before going to the server.

Reply:
done